utils/common/src/main/java/org/apache/brooklyn/util/javalang/coerce/EnumTypeCoercions.java

```diff
@@ -70,11 +70,11 @@ public E apply(String input) {
     }
     
     @SuppressWarnings({ "unchecked", "rawtypes" })
-    public static <T> Maybe<T> tryCoerceUntyped(String input, Class<T> targetType) {
+    public static Maybe tryCoerceUntyped(String input, Class targetType) {
         if (input==null) return null;
         if (targetType==null) return Maybe.absent("Null enum type");
         if (!targetType.isEnum()) return Maybe.absent("Type '"+targetType+"' is not an enum");
-        return tryCoerce(input, (Class<Enum>)targetType);
+        return tryCoerce(input, targetType);
```

Comment:
I'd keep the generics as is and add a cast to `(Maybe<T>) ` of the return value (insert before `tryCoerce`).
Credit to Eclipse's quick fix for this one :).